Assets/LeapMotion/Core/Scripts/LeapImageRetriever.cs

```diff
@@ -337,7 +348,21 @@ public class EyeTextureData {
 
     private void onImageReady(object sender, ImageEventArgs args) {
       Image image = args.image;
-      _imageQueue.TryEnqueue(image);
+
+      if (!_imageQueue.TryEnqueue(image)) {
+        Debug.LogWarning("Image buffer filled up.  This is unexpected means images are being provided faster than " +
```

Comment:
`This is unexpected means` <-- typo. Most string output we have elsewhere has a single space after periods, can we enforce that here as well?